modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -559,7 +559,7 @@ public void onClick(final DialogInterface dialog, final int id) {
     }
 
     @ReactMethod
-    public void sendLogs(final String dbJson, final Callback callback) {
+    public void sendLogs(final String dbJson, final String jsLogs, final Callback callback) {
```

Comment:
just noticed WIP here, but I'll still mark that it isn't used now

Reply:
Probably it will not be used but still will be here as part of API. On Android we have an alternative way of retrieving logs